modules/social_features/social_content_block/src/ContentBuilder.php

```diff
@@ -188,15 +195,18 @@ public function build($entity_type_id, $entity_id) {
     $block_content = $this->entityTypeManager->getStorage('block_content')
       ->load($entity_id);
 
-    if (!$block_content instanceof BlockContentInterface || $block_content->bundle() !== 'custom_content_list') {
+    if (
+      !$block_content instanceof BlockContentInterface ||
+      $block_content->bundle() !== 'custom_content_list'
+    ) {
       return [];
     }
 
     $data = [
       '#theme' => 'social_content_block',
       '#title' => $block_content->label(),
-      '#subtitle' => $block_content->getFieldValue('field_subtitle', 'value'),
-      '#topics' => $this->getTopics($block_content),
+      '#subtitle' => $block_content->field_subtitle->value,
+      '#topics' => $this->getEntities($block_content),
```

Comment:
If entities are changed from topics, should we change the key name too?

Reply:
What do you mean?